### ISSUE TITLE
Break dependency on MS Windows

### ISSUE THREAD

**Issue by eyalroz** [SunsetQuest/CudaPAD#5]
This seems like a great app (which BTW NVIDIA itself should have really released itself independently of the source view in NSight Compute). However - it is limited to a single operating system (and requires a specific C++ IDE, see #4).

Would it be possible for you to break this dependency?

I realize that is a rather tall order for C#, which is rather Windows-centric, so this might be a disguised request to port this to another language, but it would be what I would need to use CudaPAD.

**Comment by SunsetQuest** [comment]
Yeah, this is a change that needs to be made - most developers are not on windows anymore - except me =). 
I'm thinking it would be possible but how much work.  .Net 5 and core have been more independent of the OS.   I guess it would depend on .net and 3rd party items that are not universal yet and replace them.

Reviewing the code. anything with System.Forms would need to be swapped out.
 - System.Forms would need to be replaced with Microsoft.Maui 
 - System.Graphics would need to move to Microsoft.Maui.Graphics
 - ScintillaNET source is very tied to windows.forms, so that is not usable.
